src/StructureElementConnectivityRestraint.cpp

```diff
@@ -91,8 +135,22 @@ float StructureElementConnectivityRestraint::get_mean_distance_per_residue() con
   int nres = get_number_of_residues();
  
   float mean;
-  if (nres > 29) { mean = nhh_means_[29];
-  } else { mean = nhh_means_[nres];};
+
+    int sset;
+    Floats ses_ss_prob_a = atom::SecondaryStructureResidue(get_model(), a_).get_all_probabilities();
+    Floats ses_ss_prob_b = atom::SecondaryStructureResidue(get_model(), b_).get_all_probabilities();
+    if (ses_ss_prob_a[0]==1 && ses_ss_prob_b[0]==1) {
+            sset = 0;
```

Comment:
Minor point: very wonky indentation here. IMP standard is 2-space indents, no tabs: https://integrativemodeling.org/nightly/doc/manual/code_conventions.html